README.md

```diff
@@ -148,6 +148,11 @@ enum PotApplicationStatus {
 }
 ```
 
+#### ✅ Get registrations to lists by account: `GET /accounts/{ACCOUNT_ID}/registrations` (paginated)
+
+Can specify status to filter by using `status` query param if desired, e.g. `status=Approved`
+Can also specify project category to filter by using `category` query param if desired, e.g. `category=Education`
```

Comment:
what is this? why would I want to filter by category when i'm getting registrations for a single project?

Reply:
i thought projects could apply to more than one list?